lib/helpers/time.js

```diff
@@ -53,6 +85,19 @@ function getDisplayFormat(time) {
   return moment.tz(time, timezone).format('h:mm a z');
 }
 
+function getDisplayFormatForDays(days) {
+  if(days.length === 5) {
+    return 'all weekdays';
+  }
+
+  return days.reduce((previous, current, index) => {
+    if(index == days.length - 1 && days.length > 1) {
+      return `${previous} and ${current}`;
+    }
+    return `${previous}${previous.length > 0 ? ', ' : ''}${current}`;
+  }, '');
```

Comment:
This can be simplified a bit, what do you think?
```javascript
if (days.length <= 1) {
  return days[0] || 'no days'; // not sure about this
}

let last = days[-1];
let rest = days[:-1];

return rest.join(', ') + ' and ' + last;
```